### PR TITLE
Add files and add submission of Application Reviews

### PR DESCRIPTION
**Pull request by dhaselhan**
* Move front end files to their own module

### CODE REVIEW

File: services/apps/portal/src/application-review/application-review.service.ts [hunk 1]
```diff
@@ -103,4 +124,57 @@ export class ApplicationReviewService {
...
+      ) {
+        throw new BaseServiceException('Zoning information not complete');
+      }
+    }
+
+    if (applicationReview.isAuthorized === null) {
+      throw new BaseServiceException('Review authorization needs to be set');
+    }
+
+    //TODO: Where do we verify the documents are uploaded?
```
Comment (mhuseinov) [inline services/apps/portal/src/application-review/application-review.service.ts:176]: I thought that verification of documents would check that at least one document of each required type is attached. However, have no idea what is required other than certificate of title

File: services/apps/portal/src/application/application-document/application-document.entity.ts [hunk 2]
```diff
@@ -11,8 +11,17 @@ import {
 import { Document } from '../../document/document.entity';
 import { Application } from '../application.entity';
 
-export const DOCUMENT_TYPES = ['certificateOfTitle'] as const;
-export type DOCUMENT_TYPE = 'certificateOfTitle';
+export const DOCUMENT_TYPES = [
```
Comment (mhuseinov) [inline services/apps/portal/src/application/application-document/application-document.entity.ts:14]: probably need a ticket to move this types to code tables on ALCS

**Review by mhuseinov — APPROVED** [review]

File: services/apps/portal/src/application-review/application-review.service.ts (hunk 1 above)
Comment (dhaselhan) [reply services/apps/portal/src/application-review/application-review.service.ts]: Added it here, I was on the edge of not passing the application to the application review service, but I think its easier to have all the validation checks done in a single place.

File: services/apps/portal/src/application/application-document/application-document.entity.ts (hunk 2 above)
Comment (dhaselhan) [reply services/apps/portal/src/application/application-document/application-document.entity.ts:14]: Probably a good idea, its not the nicest method currently

**Pull request merged**